src/js/support/mobile/widget/Datetimepicker.js

```diff
@@ -428,7 +428,7 @@
 								values = getCalendar().months.names.slice();
 								break;
 						}
-						if (values.length === 13) { // @TODO Lunar calendar support
+						if (values && values.length === 13) { // @TODO Lunar calendar support
```

Comment:
I think it is not solution of the issue.
We need "default" case in switch statement in where `values` will be assign

Reply:
done